lib/sass.js

```diff
@@ -31,9 +46,13 @@ export async function buildCSS(inputs) {
         sourceMap: sourcemapPath,
       });
 
-      const cssProcessor = postcss([autoprefixer()]);
+      const cssProcessor = postcss([
+        tailwindcss(tailwindConfig),
+        autoprefixer(),
+      ]);
+
       const postcssResult = await cssProcessor.process(sassResult.css, {
-        from: output,
+        from: input,
```

Comment:
I changed this to fix an error if the output file does not already exist. See notes in commit.